### ISSUE TITLE
libtorch : latest version for cpu , torch::nn::functional::pad  not found

### ISSUE THREAD

**Issue by EsmaeilFarhang** [pytorch/pytorch#30601]
## To Reproduce
 - Libtorch Version (1.3.1 stable debug version):
 - OS (windows 10):
 - How you installed Libtorch ( source):
 - Build command you used (visual stdio):
 - cpp version:c++11
 - cpu version:

## Additional context
acording to  libtorch documentation FUNCTION TORCH::NN::FUNCTIONAL::PAD should be 
in this directory ("torch/csrc/api/include/torch/nn/functional/") but   some header files such as  padding.h and ... dont exist

cc @yf225

**Comment by peterjc123** [comment]
This seems to be fixed in master because I can see them in the log.
```
	  New Dir         16	C:\w\1\s\windows\pytorch\torch\include\torch\csrc\api\include\torch\nn\functional\
	    New File  		   10126	activation.h
  0%  
100%  
	    New File  		    1519	batchnorm.h
  0%  
100%  
	    New File  		    4768	conv.h
  0%  
100%  
	    New File  		    1571	distance.h
  0%  
100%  
	    New File  		    3389	dropout.h
  0%  
100%  
	    New File  		    5298	embedding.h
  0%  
100%  
	    New File  		    1951	fold.h
  0%  
100%  
	    New File  		     919	instancenorm.h
  0%  
100%  
	    New File  		     835	linear.h
  0%  
100%  
	    New File  		   19727	loss.h
  0%  
100%  
	    New File  		    3935	normalization.h
  0%  
100%  
	    New File  		    3803	padding.h
  0%  
100%  
	    New File  		     570	pixelshuffle.h
  0%  
100%  
	    New File  		   21925	pooling.h
  0%  
100%  
	    New File  		    5131	upsampling.h
  0%  
100%  
	    New File  		    3009	vision.h
  0%  
```

**Comment by EsmaeilFarhang** [comment]
thank you, you right. in Preview (Nightly) version this exists. but in a stable version for ubuntu and windows, some header files don't exist.

**Comment by pietern** [comment]
@EsmaeilFarhang Is this file mentioned in the 1.3.1 or master documentation?

**Comment by yf225** [comment]
@EsmaeilFarhang Currently the libtorch documentation always points to the preview (nightly) version, and we will change it to be versioned (e.g. v1.2.0 / v1.3.1 / ...) in the future when the C++ frontend becomes more stable.

`torch::nn::functional::pad` will be included in our next stable release v1.4.0 in the coming weeks.

**Comment by EsmaeilFarhang** [comment]
> @EsmaeilFarhang Is this file mentioned in the 1.3.1 or master documentation?

this files is mentioned in the master documentation.
